platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/config/MerkleDbConfig.java

```diff
@@ -57,7 +65,11 @@
 @ConfigData("merkleDb")
 public record MerkleDbConfig(
         @Positive @ConfigProperty(defaultValue = "500000000") long maxNumOfKeys,
-        @Min(0) @ConfigProperty(defaultValue = "8388608") long hashesRamToDiskThreshold,
+        @Positive @ConfigProperty(defaultValue = "" + 4_000_000_000L) long size,
+        @Min(0) @ConfigProperty(defaultValue = "" + 8388608L) long hashesRamToDiskThreshold,
```

Comment:
`"" + 8388608L` is "8388608" so the change is confusing and potentially misleading.

Reply:
This change is reverted now